.evergreen/config.yml

```diff
@@ -518,6 +518,33 @@ functions:
           PKG_CONFIG_PATH=$PKG_CONFIG_PATH \
           LD_LIBRARY_PATH=$LD_LIBRARY_PATH
 
+  run-load-balancer-tests:
```

Comment:
The script here is mostly a copy/paste of the [run-tests](https://github.com/mongodb/mongo-go-driver/blob/6c11b22eb17e572706b861bc6535658318e25d35/.evergreen/config.yml#L279) function.